### ISSUE TITLE
esp not working through walls 1.21

### ISSUE THREAD

**Issue by Foxyling** [gavinsmod/gavinsextensiblemod#77]
the esp doesnt work through walls for me atleast
i enable the esp and it works and all but when i go behind a wall the esp doesnt go through

**Comment by GT3CH1** [comment]
Hi,
Are you using any other mods? If so, please attach a list. Some things like shaders and some performance mods prevent the ESPs from being rendered correctly.

Thanks!

**Comment by Foxyling** [comment]
i am using mods with it 
the list is:
3d skin layers
cloth config api
appleskin
dynamicfps
entity culling
essential mod
fabric language kotlin
lithium
mod menu
player health indicators
replay mod
shulker box tooltip
sodium
xaero's minimap + world  map
ukulib
seedcrackerX

**Comment by Foxyling** [comment]
also im using feather client

**Comment by GT3CH1** [comment]
Which ESP are you enabling? I can see several mods in that list that would
prevent multiple ESPs from working.

**Comment by Foxyling** [comment]
ive done player and chest esp and they dont work

**Comment by GT3CH1** [comment]
Ok. I will try to reproduce the issue, but that may take a few days to do
so at the moment due to time constraints.

**Comment by Foxyling** [comment]
if u could find the issue and solve it that would be helpful
ive done the same thing on curseforge and it does work, i do believe it is a feather client thing
thanks for the help.

**Comment by GT3CH1** [comment]
Yeah I would believe so - would you try deactivating that mod? Just
drag+drop it out of your mods folder temporarily

**Comment by Foxyling** [comment]
feather client is an actual minecraft mod client / launcher and its got its own settings and stuff

**Comment by GT3CH1** [comment]
Ok. Would you try removing mods one by one until you notice that ESP works
again? That would be fantastic. Thanks!

**Comment by GT3CH1** [comment]
Unfortunately, I am unable to find the source for the Feather client, so I am unable to write anything that would help fix this issue. If you are able to find the source, please let me know and link it here so I can investigate it. Thanks!

**Comment by GT3CH1** [comment]
I'm re-opening this issue - I have written a fix that works with sodium, and quite possibly the feather client.

Please download the file `gavinsmod-1.4.8rc2+1.21.1.jar` from the pre-release found here: [https://github.com/gavinsmod/gavinsextensiblemod/releases/tag/v1.4.8-rc2](https://github.com/gavinsmod/gavinsextensiblemod/releases/tag/v1.4.8-rc2)

Let me know if you find anything weird that needs improving.
